pkg/storage/postgresql/ticketserver.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2023 Adevinta
+*/
+
+package postgresql
+
+import (
+	"github.com/adevinta/vulcan-tracker/pkg/model"
+)
+
+// FindServerConf retrieves a ticket tracker configuration from a postgres database.
+func (db *PostgresStore) FindServerConf(serverID string) (model.TrackerConfig, error) {
+
```

Comment:
Why this empty line at the beginning of the function?

Reply:
Fixed